src/index.html

```diff
@@ -1,14 +1,36 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <ul class="clock">
+        <li class="clock__number clock__number--number--1"><span>1</span></li>
+        <li class="clock__number clock__number--number--2"><span>2</span></li>
```

Comment:
```suggestion
        <li class="clock__number clock__number--1"><span>1</span></li>
        <li class="clock__number clock__number--2"><span>2</span></li>
```
Do you really need to duplicate number?